plutus-core/executables/plutus/Mode/PrintBuiltins.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE ViewPatterns #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE DataKinds           #-}
+module Mode.PrintBuiltins
+    ( runPrintBuiltins
+    ) where
+
+import PlutusPrelude
+import PlutusCore qualified as PLC
+import PlutusCore.Builtin qualified as PLC
+import PlutusCore.Normalize (normalizeType)
+
+import Prettyprinter qualified as PP
+import Text.Printf
+import Data.Proxy (Proxy (..))
+import Data.List (intercalate)
+import GHC.TypeLits (symbolVal)
+
+runPrintBuiltins :: IO ()
+runPrintBuiltins = portedImpl
+
+-- following is PORTED from the older executable
+
+portedImpl :: IO ()
+portedImpl = do
+    -- MAYBE: categorize the builtins by Plutus Version introduced. Would require dependency
+    -- upon plutus-ledger-api
+    let builtins = enumerate @PLC.DefaultFun
+    mapM_
+      (\x -> putStr (printf "%-35s: %s\n" (show $ PP.pretty x) (show $ getSignature x)))
+      builtins
```

Comment:
Nit: I'd write it as `for_ builtins $ \x -> <...>`.

Reply:
It is ported code, I didn't bother editing at all. It is a copy-paste